pkg/kademlia/kademlia_test.go

```diff
@@ -258,7 +257,7 @@ func TestFindNear(t *testing.T) {
 		require.NoError(t, err)
 		return *n
 	}
-	nodeIDA := newNode("AAAAA", 1, 4)
+	var nodeIDA = newNode("AAAAA", 1, 4)
```

Comment:
I'm just wondering why this is much better than using `:=`?

Reply:
Goland recommends the change but I don't know which is objectively better. It may be that Goland defaults to var since := has some exceptions but don't apply here.

Reply:
Turns out @egonelbre says the original is more idomatic actually... I can change them back